NodeManager.h

```diff
@@ -972,6 +972,8 @@ class SensorDs18b20: public Sensor {
 class SensorBH1750: public Sensor {
   public:
     SensorBH1750(NodeManager* node_manager, int child_id);
+    // set sensor reading mode, e.g. BH1750_ONE_TIME_HIGH_RES_MODE
```

Comment:
Would you mind both here and in the documentation making this comment looking like the following:
`// [101] set sensor reading mode, e.g. BH1750_ONE_TIME_HIGH_RES_MODE`
So to make explicit the id to use for the remote api like the other functions. Thanks!